src/interfaces/set-configuration-interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface SetConfigurationIF  {
```

Comment:
Or maybe call it something generic that can be reused?

Reply:
Good idea.  I'll rename it 'KeyValueIF'